src/main/java/core/mate/academy/impl/BulldozerProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.impl;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private List<Bulldozer> list = new ArrayList<>();
```

Comment:
The field 'list' should have an access modifier to encapsulate its access. Consider using 'private' to restrict direct access from outside of the class.